.github/workflows/build-test.yml

```diff
@@ -17,6 +17,7 @@ jobs:
     name: Code Style
     uses: playframework/.github/.github/workflows/cmd.yml@v4
     with:
+      env: JAVA_TOOL_OPTIONS='--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED'
```

Comment:
fyi: only works with single quotes (because of multiple `=` in one line): https://github.com/github/docs/pull/21582